### PR TITLE
Feature calc

### PR DESCRIPTION
**Pull request by ViniciusSossela**
screen shot 2018-01-10 at 15 13 19

### CODE REVIEW

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java [hunk 1]
```diff
@@ -23,11 +17,10 @@
...
     LayoutInflater inflater;
-    View layoutView;
-    View verticalSeekBarBackground;
-    View verticalSeekBarThumb;
-    float dX = 0, dY = 0;
-    int marginTop = 0;
+    View layoutView, verticalSeekBarBackground, verticalSeekBarThumb;
+    VerticalSeekBarListener listener;
+    float dY = 0;
+    int marginTop = 0, valor = 0, step = 0, calculatedValue = 0;
```
Comment (cleberhenriques) [inline verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:23]: All properties have english names, but the `valor` is in portuguese. Can you fix that?

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java [hunk 2]
```diff
@@ -37,88 +30,83 @@ public VerticalSeekBar(Context context, @Nullable AttributeSet attrs) {
...
                 0, 0);
+        loadBackgroundAttr(attributes);
+        loadThumbAttr(attributes);
+        loadValueAttr(attributes);
+        loadStepAttr(attributes);
+        setupBackgroundMarginTopAttr(attributes);
+    }
 
+    private void loadValueAttr(TypedArray attributes) {
+        valor = attributes.getInteger(R.styleable.VerticalSeekBar_seekbar_value, 500);
```
Comment (cleberhenriques) [inline verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:56]: What is 500? Can you put that in a constant?

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java [hunk 3]
```diff
@@ -37,88 +30,83 @@ public VerticalSeekBar(Context context, @Nullable AttributeSet attrs) {
...
+    }
 
+    private void loadStepAttr(TypedArray attributes) {
+        step = attributes.getInteger(R.styleable.VerticalSeekBar_seekbar_step, 25);
```
Comment (cleberhenriques) [inline verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:60]: What is 25? Can you put that in a constant?

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java (hunk 2 above)
Comment (ViniciusSossela) [reply verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:56]: São valores mágicos,, só deus sabe o que representam...

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java (hunk 1 above)
Comment (ViniciusSossela) [reply verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:23]: valor has no english word... fixing :)

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java (hunk 2 above)
Comment (ViniciusSossela) [reply verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:56]: ok, fixing

File: verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java (hunk 3 above)
Comment (ViniciusSossela) [reply verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java:60]: same as above

**Pull request merged**